src/main/java/nl/tudelft/ewi/devhub/server/database/controllers/AssignedTAs.java

```diff
@@ -0,0 +1,51 @@
+package nl.tudelft.ewi.devhub.server.database.controllers;
+
+import com.google.inject.Inject;
+import com.mysema.query.jpa.JPASubQuery;
+import nl.tudelft.ewi.devhub.server.database.entities.*;
+
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.mysema.query.group.GroupBy.groupBy;
+import static com.mysema.query.group.GroupBy.list;
+import static java.util.Comparator.comparing;
+import static nl.tudelft.ewi.devhub.server.database.entities.QAssignedTA.assignedTA;
+import static nl.tudelft.ewi.devhub.server.database.entities.QDelivery.delivery;
+
+/**
+ * Created by sayra on 07/06/2017.
+ */
+public class AssignedTAs extends Controller<AssignedTA> {
```

Comment:
Testing the query is indeed a good idea. I'll have a look at this after isolating the subquery since testing the queries separately would be more convenient.